mkosi

```diff
@@ -341,16 +419,26 @@ def mount_image(args, workspace, loopdev):
     root = os.path.join(workspace, "root")
 
     if args.output_format != OutputFormat.raw_squashfs:
-        mount_loop(args, loopdev, args.root_partno, root)
+        root_dev = partition(loopdev, args.root_partno)
+
+        if "root" in luks_keys:
+            root_dev = luks_open(root_dev, luks_keys["root"])
+        mount_loop(args, root_dev, root)
```

Comment:
I think mount_loops_with_luks() would be good to have too, that wraps mount_loops() and takes the key as extra argument applying it if it is set, and if it s None not.